### PR TITLE
Added functionality to enable or disable Google Fonts

### PR DESCRIPTION
**Pull request by vhbelvadi**
This edit keeps Google Fonts enabled by default, so it does not change how the theme works or looks out of the box, but adds a site parameter to disable (or explicitly enable) Google Fonts for those who want to customise fonts and disable Google Fonts without replicating `head-html.html` in their root `layouts/partials` directory.

### CODE REVIEW

**Comment by alex-shpak** [comment]
Good 👍 

Maybe we should go further? Fonts are the only thing that loaded from internet, maybe we should put font as static files and they make them customisable.
Then theme will be intranet-compatible.

File: layouts/partials/docs/html-head.html [hunk 1]
```diff
@@ -3,7 +3,10 @@
 <title>{{- template "title" . }} | {{ .Site.Title -}}</title>
 
 <!-- Fonts for the theme -->
-<link href="https://fonts.googleapis.com/css?family=Oxygen|Oxygen+Mono:300,400,700" rel="stylesheet">
+{{ with .Site.Params.googleFonts | default "true" }}
```
Comment (alex-shpak) [inline layouts/partials/docs/html-head.html]: I think you can do
```{{ if default true .Site.Params.googleFonts }}

{{ end }}
```

Comment (vhbelvadi) [reply layouts/partials/docs/html-head.html]: Agreed. This is better.

**Comment by vhbelvadi** [comment]
I second the idea. Making the theme intranet-compatible was partly what made me work on disabling Google Fonts. If we host fonts locally it should be much, much better.

What exactly do you mean by customisable? Do you mean enabling/disabling static font files or choosing fonts? In the latter case I do not think we should keep several static files. We can enable choosing any Google Font via config since it'll only involve changing the a couple of words in the url. But again, it won't work 100% offline.

**Comment by alex-shpak** [comment]
I thought like this:
1. Put fonts into repo, to `static` folder
2. Add `_fonts.scss` with importing these fonts, maybe put `body { font-face: ... }` there as well
3. Import that file to `book.scss`

Then you can override `_fonts.scss` in your website and put any fonts you want

**Comment by vhbelvadi** [comment]
**Update** New commits to load Oxygen locally from `static` via `_fonts.scss`.

However, instead of loading it in `body.scss` we continue to load it in `html-head` (as before) based on the googleFonts parameter set in config.

**Current behaviour:**
1. Theme loads Oxygen locally by default
2. User can turn off Oxygen via `config` to use default font or
3. User can load their own font via `_fonts.scss` in root to override Oxygen regardless of `config`

By loading `_fonts.scss` in `html-head` both 2 and 3 are possible. Else 2 would not be possible: i.e. if we stop conditional loading of `_fonts.scss` from `html-head` a user who wants their site based entirely on default fonts cannot achieve that easily, now they can with a `config` parameter.

**Comment by alex-shpak** [comment]
Nice.
I feel like variable is redundant, one can override `_fonts.scss` in site to disable default font.
I think less variables as better :)

**Comment by alex-shpak** [comment]
Maybe also move font files to `static/font` 🤔

**Comment by vhbelvadi** [comment]
On second thought maybe you're right. Since we load fonts locally it shouldn't be an issue to use Oxygen by default. I'll get rid of the separate parameter.

**Comment by vhbelvadi** [comment]
All done.

1. Theme `body.scss` now imports `_fonts.scss`
2. Nothing loaded in `html-head`
3. Fonts moved to `static/fonts`
4. Updated readme under ‘extra customisation’ describing the use of `_fonts.scss`

Also checked the overriding mechanism with a sample theme using appropriate replacement files in the theme root directory—everything works as expected. It should be OK to merge now.

**Comment by alex-shpak** [comment]
👍

**Comment by alex-shpak** [comment]
@vhbelvadi 
Just FYI, I changed it a bit
https://github.com/alex-shpak/hugo-book/commit/612814aa595fd31df6cab4d4ace917257d62ca3f

**Pull request merged**